lib/serializer.js

```diff
@@ -1,9 +1,8 @@
 'use strict';
 
-const JSONAPISerializer = require('jsonapi-serializer').Serializer;
+var JSONAPISerializer = require('jsonapi-serializer').Serializer;
 
 module.exports = function (modelName, data) {
-  let sails = this.req._sails;
```

Comment:
This should not be removed. It's used several times. Causing your tests to fail.